### ISSUE TITLE
Quick start tutorial not working with Java 8 

### ISSUE THREAD

**Issue by soisik** [lsc-project/lsc#2]
Build environment : Apache Maven 3.0.2 & Java 1.6
Runtime environment : Java 1.8
Quick start tutorial : http://lsc-project.org/wiki/documentation/2.0/sample.

### Problem
Java 6 or 7 is required to get the demo working. With Java 8, we have this error on server startup :
```
Exception in thread "main" java.lang.ExceptionInInitializerError: A security class cannot be found in this JVM because of the following reason: sun.security.x509.CertAndKeyGen
at org.opends.server.util.Platform$PlatformIMPL.<clinit>(Platform.java:127)
```
### Solution
According to https://forgerock.org/topic/opendj-2-6-does-not-compatible-with-java-8/, for java 8 support, OpenDJ should be upgraded to at least 2.6.2. Current version is 2.4.5.
This is done by :
- changing the version in the pom,
- updating server conf under src/test/resources/etc/config/
- updating sample/hsqldb/etc/test.ldif use to initialize root dn

### Notes :
- OpenDJ being also used in junit tests, this fix is needed for running a mvn build with java 8.
- OpenDJ is only used in demo and junit tests, so this fix should have no impact on core functionalities

**Comment by soisik** [comment]
Fixed, will be available in 2.1.4

**Comment by soisik** [comment]
Fixed by merge in 2.2.0
